thirdFile.txt

```diff
@@ -1,2 +1,2 @@
 Third File
-Author - Saion
\ No newline at end of file
+Author - Aditya
```

Comment:
Add full name instead of just first name

Reply:
Done.